apis/v1beta1/providerconfig_types.go

```diff
@@ -25,6 +25,8 @@ import (
 // A ProviderConfigSpec defines the desired state of a ProviderConfig.
 type ProviderConfigSpec struct {
 	v1alpha1.ProviderConfigSpec `json:",inline"`
+	// Region for IBM Cloud API
+	Region string `json:"region,omitempty"`
```

Comment:
Is region actually scoped to your IBM cloud credentials or is it just required for every resource type?

Reply:
Region is needed to acquire the token used in all API calls, then each service could also have a region to indicate where it gets deployed. The two regions might be different (I might acquire a token in `us-south` and provision a service in `ap-south`).

Reply:
@pdettori makes sense :+1: Should it be required here then?

Reply:
I think it can be optional as there is a default (`ibm-south` ) when it is not specified.